package.json

```diff
@@ -375,8 +375,8 @@
     "redux-saga": "^1.1.3",
     "redux-thunk": "^2.3.0",
     "redux-thunks": "^1.0.0",
-    "remark-stringify": "^9.0.0",
     "regenerator-runtime": "^0.13.3",
```

Comment:
I did `yarn add @types/lz-string --dev` and it rearranged a couple things in here and yarn.lock.